code/style.css

```diff
@@ -64,34 +71,38 @@ select {
   width: 135px;
   height: 165px;
   overflow: hidden;
-  border: 3px solid var(--secondary);
-  border-radius: 4px;
+  border-radius: 6px;
   margin: 3px;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: space-between;
+  background-color: white;
+  background-image: linear-gradient(rgba(60, 125, 209, 0.8), rgba(44, 95, 190, 0.8)),
+  url("images/card-background1.png");
 }
 
 .card p {
   text-align: center;
   margin-bottom: 0;
   font-size: 16px;
   font-weight: 700;
+  color: #303599;
 }
 
 .card .guess {
   display: none;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  color: white;
+  color: #303599;
   text-align: center;
   height: inherit;
 }
 
```

Comment:
Love that you have done it like the old (well sort of old) board game! With the colors and that the background on the cards is yellow when you hover it.